app/client-graphql-test/page.tsx

```diff
@@ -13,6 +11,11 @@ query ContributorRoles{
     }
 }`;
 
+type ContributorRoles = {
```

Comment:
Could you setup codegen instead that auto-generates the types for you based on the GraphQL schema definition in the backend?
example `codegen.ts`
```
import type { CodegenConfig } from "@graphql-codegen/cli";

const config: CodegenConfig = {
  schema: "../dmsp_backend_prototype/src/schemas/*.ts",
  generates: {
    "./src/types.ts": {
      plugins: ["typescript", "typescript-resolvers"],
      config: {
        contextType: "../dmsp_backend_prototype/src/context#MyContext",
        enumsAsTypes: true,
      },
    },
  },
};

export default config;
```

Reply:
@briri Yes that's a good idea. I'll create a separate ticket for that.

Reply:
@briri I created this ticket to add the use of graphql-codegen: https://github.com/CDLUC3/dmsp_frontend_prototype/issues/88

Also, I added a "Loading..." message for screen readers in the TypeAheads.